interfaces/HO-Portal/src/assets/styles/ngx-datatable/_ngx-datatable.component.scss

```diff
@@ -1,7 +1,6 @@
 // Source, see: https://github.com/swimlane/ngx-datatable/blob/16.1.1/projects/swimlane/ngx-datatable/src/lib/components/datatable.component.scss
 .ngx-datatable {
   display: block;
-  overflow: hidden;
```

Comment:
Why this specific change?
I think we should update this whole file from v16.1.1 (as stated in the top comment, to v20.1.0.(the version we actually use)

And apply any overrides that are necessary _on top of that_, instead of changing the "out of the box behaviour" of the package.

Reply:
I've updated the file